test/spec/modules/adponeBidAdapter_spec.js

```diff
@@ -110,122 +110,102 @@ describe('adponeBidAdapter', function () {
       expect(spec.isBidRequestValid(invalidBid)).to.be.false;
     });
   });
-});
-
-describe('interpretResponse', function () {
-  let serverResponse;
-  let bidRequest = { data: {id: '1234'} };
-
-  beforeEach(function () {
-    serverResponse = {
-      body: {
-        id: '2579e20c0bb89',
-        seatbid: [
-          {
-            bid: [
-              {
-                id: '613673EF-A07C-4486-8EE9-3FC71A7DC73D',
-                impid: '2579e20c0bb89_0',
-                price: 1,
-                adm: '<html><a href="https://www.adpone.com" target="_blank"><img src ="https://placehold.it/300x250" /></a></html>',
-                adomain: [
-                  'www.addomain.com'
-                ],
-                iurl: 'https://localhost11',
-                crid: 'creative111',
-                h: 250,
-                w: 300,
-                ext: {
-                  dspid: 6
+  describe('interpretResponse', function () {
+    let serverResponse;
+    let bidRequest = { data: {id: '1234'} };
+
+    beforeEach(function () {
+      serverResponse = {
+        body: {
+          id: '2579e20c0bb89',
+          seatbid: [
+            {
+              bid: [
+                {
+                  id: '613673EF-A07C-4486-8EE9-3FC71A7DC73D',
+                  impid: '2579e20c0bb89_0',
+                  price: 1,
+                  adm: '<html><a href="https://www.adpone.com" target="_blank"><img src ="https://placehold.it/300x250" /></a></html>',
+                  adomain: [
+                    'www.addomain.com'
```

Comment:
Please set bid.meta.advertiserDomains

Reply:
Hey, I have removed this property. Is that ok?

Reply:
Just add it to bid object. Search the repo for meta.advertiserDomains for examples how